js/components/viewControls.js

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+var DirectionControl = require('./directionControl');
+var classnames = require('classnames');
+
+var ViewControls = React.createClass({
```

Comment:
Can we change this to ViewControlsVr or something similar? To be clear that it is a VR related functionality

Reply:
\+